source/clear-linux/guides/telemetrics/telemetry-enable.rst

```diff
@@ -0,0 +1,114 @@
+.. _telemetry-enable:
+
+Install and enable telemetry in |CL-ATTR|
```

Comment:
Please revise page title to simply identify the topic and remove product name e.g. "Enable telemetry"